relion/protocols/protocol_compress_movies.py

```diff
@@ -0,0 +1,165 @@
+# ******************************************************************************
+# *
+# * Authors:     J.M. De la Rosa Trevin (delarosatrevin@scilifelab.se) [1]
+# *
+# * [1] SciLifeLab, Stockholm University
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 3 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# ******************************************************************************
+
+import os
+
+import pyworkflow.protocol.params as params
+import pyworkflow.utils as pwutils
+from pwem.protocols import ProtAlignMovies
+from pwem.objects import Image, MovieAlignment
+from pyworkflow.protocol import STEPS_PARALLEL
+
+import relion
+
+
+class ProtRelionCompressMovies(ProtAlignMovies):
+    """
+    Using *relion_convert_to_tiff to compress a set of movies
+    or to estimate the gain that can be used for better compression.
+    """
+
+    _label = 'compress movies'
+
+    OP_COMPRESS = 0
+    OP_ESTIMATE = 1
+
+    def __init__(self, **kwargs):
+
+        ProtAlignMovies.__init__(self, **kwargs)
+        self.stepsExecutionMode = STEPS_PARALLEL
+
+    def _getConvertExtension(self, filename):
+        """ Check whether it is needed to convert to .mrc or not """
+        ext = pwutils.getExt(filename).lower()
+        return None if ext in ['.mrc', '.mrcs', '.tiff', '.tif'] else 'mrc'
```

Comment:
add tif check as above

Reply:
Not completely sure, but I guess tiff images can also be compressed and written as another tiff file.